src/utils/utilities.ts

```diff
@@ -0,0 +1,8 @@
+import { CONSTANTS } from "./contants";
+
+export const getDirectoriesToExclude = (excludedDirectoriesInput: Array<string>): Array<string> => {
+  const cleansedInput = excludedDirectoriesInput.map((pattern) => pattern.trim());
+  return removeDuplicates([...cleansedInput, ...CONSTANTS.SOOS.DEFAULT_DIRECTORIES_TO_EXCLUDE]);
```

Comment:
I did this intentionally, dont know if there would be a scenario where the user will want to scan something on node_modules for example.

Reply:
we should exclude them by default, but allow override